build.gradle

```diff
@@ -395,10 +414,8 @@ subprojects {
     classifier = 'tests'
   }
 
-  if (project.version != project.DEFAULT_VERSION) {
-    if (publishRegularArtifacts) {
-      publishing.configureRegularArtifactPublishing(project, projectsToPublish.get(project.path), sourceJar, testJar)
-    }
+  if (publishRegularArtifacts) {
```

Comment:
This flag is already checked on line 399, let's combine both blocks and eliminate the local variable.

Reply:
Done. Thanks for the suggestion